samples/msgext-northwind-inventory-csharp/README.md

```diff
@@ -0,0 +1,100 @@
+---
+page_type: sample
+description: This sample implements a Teams message extension that can be used as a plugin for Microsoft Copilot for Microsoft 365. The message extension allows users to query the Northwind Database.
+products:
+- office-teams
+- copilot-m365
+languages:
+- dotnet
+- csharp
+---
+
+# Northwind inventory message extension sample
+
+![License.](https://img.shields.io/badge/license-MIT-green.svg)
+
+This sample implements a Teams message extension that can be used as a plugin for Microsoft Copilot for Microsoft 365. The message extension allows users to query the [Northwind Database](https://learn.microsoft.com/dotnet/framework/data/adonet/sql/linq/downloading-sample-databases).
+
+![Screenshot of the sample extension working in Copilot in Microsoft Teams](./lab/images/03-03a-response-on-chai.png)
+
+## Version history
+
+Version|Manifest version|Date|Author|Comments
+-------|--|--|----|--------
+1.0|1.16|November 15, 2023 |Jegadeesh V <br/> Wajeed Shaikh|Initial release for Csharp labs
+
+## Prerequisites
+
+- [Visual Studio 2022 17.10+](https://visualstudio.microsoft.com)
+- [Azure subscription](https://portal.azure.com)
+- [Teams Toolkit for Visual Studio](https://learn.microsoft.com/en-us/microsoftteams/platform/toolkit/toolkit-v4/install-teams-toolkit-vs?pivots=visual-studio-v17-7)
+- You will need a Microsoft work or school account with [permissions to upload custom Teams applications](https://learn.microsoft.com/microsoftteams/platform/concepts/build-and-test/prepare-your-o365-tenant#enable-custom-teams-apps-and-turn-on-custom-app-uploading). The account will also need a Microsoft Copilot for Microsoft 365 license to use the extension in Copilot.
+- You will need to create [local Azure Storage](https://learn.microsoft.com/en-us/azure/storage/common/storage-use-azurite?tabs=visual-studio%2Cblob-storage).
+
+## Setup local Azure Storage
```

Comment:
Visual Studio 2022 has Azure Storage emulation built in, we should look to use this instead of introducing a dependency on node and the Azurite npm package. See [Use the Azurite emulator for local Azure Storage development: Running Azurite from an ASP.NET project](https://learn.microsoft.com/en-gb/azure/storage/common/storage-use-azurite?tabs=visual-studio%2Cblob-storage#running-azurite-from-an-aspnet-project)

Reply:
Fixed